### PR TITLE
M2-1544: Dataviz - Bar chart

### PR DESCRIPTION
**Pull request by nmialik**

### CODE REVIEW

**Review by muhurov — APPROVED** [review]

File: package.json
Comment (IhorNesterenko99) [inline package.json:1]: pls add increment of version

File: src/modules/Dashboard/features/RespondentData/RespondentDataSummary/Charts/BarChart/BarChart.tsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+export const BarChart = () => {
+  const options = {
+    responsive: true,
+    plugins: {
+      legend: {
+        labels: {
+          color: variables.palette.on_surface,
+          font: {
+            family: 'Atkinson',
+            size: 14,
```
Comment (IhorNesterenko99) [inline src/modules/Dashboard/features/RespondentData/RespondentDataSummary/Charts/BarChart/BarChart.tsx:28]: could you pls use variables.font.size.md?

File: src/modules/Dashboard/features/RespondentData/RespondentDataSummary/Charts/BarChart/BarChart.tsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        border: {
+          display: false,
+          dash: [8, 8],
+        },
+        ticks: {
+          stepSize: 2,
+          color: variables.palette.on_surface,
+          font: {
+            family: 'Atkinson',
+            size: 14,
```
Comment (IhorNesterenko99) [inline src/modules/Dashboard/features/RespondentData/RespondentDataSummary/Charts/BarChart/BarChart.tsx:56]: also here font size from variables

File: src/modules/Dashboard/features/RespondentData/RespondentDataSummary/Report/Report.tsx [hunk 3]
```diff
@@ -6,6 +6,7 @@ import { Activity } from 'redux/modules';
 import { Svg, Tooltip } from 'shared/components';
 import { useHeaderSticky } from 'shared/hooks';
 import { StyledHeadlineLarge, theme } from 'shared/styles';
+import { BarChart } from 'modules/Dashboard/features/RespondentData/RespondentDataSummary/Charts/BarChart/BarChart';
```
Comment (IhorNesterenko99) [inline src/modules/Dashboard/features/RespondentData/RespondentDataSummary/Report/Report.tsx:9]: could you pls add index.ts to BarChart and import here from index?

**Review by alex-i-e — APPROVED** [review]

File: src/modules/Dashboard/features/RespondentData/RespondentDataSummary/Charts/BarChart/BarChart.tsx (hunk 1 above)
Comment (nmialik) [reply src/modules/Dashboard/features/RespondentData/RespondentDataSummary/Charts/BarChart/BarChart.tsx:28]: variables.font.size.md is string - '1.4rem'
here should be just a number

File: src/modules/Dashboard/features/RespondentData/RespondentDataSummary/Charts/BarChart/BarChart.tsx (hunk 2 above)
Comment (nmialik) [reply src/modules/Dashboard/features/RespondentData/RespondentDataSummary/Charts/BarChart/BarChart.tsx:56]: the same

**Review by IhorNesterenko99 — APPROVED** [review]

**Pull request merged**